Gemfile.lock

```diff
@@ -77,6 +77,20 @@ GEM
       terminal-table (>= 1.5.1)
     globalid (0.4.2)
       activesupport (>= 4.2.0)
+    haml (5.2.2)
+      temple (>= 0.8.0)
+      tilt
+    haml-rails (1.0.0)
+      actionpack (>= 4.0.1)
+      activesupport (>= 4.0.1)
+      haml (>= 4.0.6, < 6.0)
+      html2haml (>= 1.0.1)
+      railties (>= 4.0.1)
+    html2haml (2.3.0)
+      erubis (~> 2.7.0)
+      haml (>= 4.0)
+      nokogiri (>= 1.6.0)
+      ruby_parser (~> 3.5)
```

Comment:
This looks like a development dependency, we can manage those separately by defining groups in the gemfile https://bundler.io/guides/groups.html

Example
```
group :development do
    html2haml (2.3.0)
    erubis (~> 2.7.0)
    haml (>= 4.0)
    nokogiri (>= 1.6.0)
    ruby_parser (~> 3.5)
end
```

But its worth considering if its worth adding to the application gemfile at all. If you are only intending to use it once, you can install the gem globally and it avoids an extra dependency.